webevd/WebEVD/web/evd.js

```diff
@@ -167,18 +167,18 @@ function FinalizeTextures(){
                                            });
         } // end for fname
     } // end for d
-
-    return mat;
```

Comment:
Unrelated to this change, but removed erronous return (`mat` isn't in scope, and the one use of this function doesn't use the return value anyways).